wezterm-gui/src/customglyph.rs

```diff
@@ -533,44 +572,63 @@ pub enum PolyCommand {
         center: BlockPoint,
         radiuses: BlockPoint,
     },
+    Circle {
+        center: BlockPoint,
+        radius: BlockCoord,
+    },
     Close,
 }
 
 impl PolyCommand {
     fn to_skia(&self, width: usize, height: usize, underline_height: f32, pb: &mut PathBuilder) {
         match self {
             Self::MoveTo(x, y) => pb.move_to(
-                x.to_pixel(width, underline_height),
-                y.to_pixel(height, underline_height),
+                x.to_pixel(width, underline_height, cmp::min(width, height)),
```

Comment:
FWIW, you should be able to do `a.min(b)` instead of `cmp::min(a, b)` for a more compact version of the same effect. I do prefer `a.min(b)` over `cmp::min`!

```suggestion
                x.to_pixel(width, underline_height, width.min(height)),
```

Reply:
Yup, I fully agree. I only get to Rust occasionally so I forgot about that shorthand.